versioned_docs/version-3.x/developer/extending/apps/architecture/app-bridge.mdx

```diff
@@ -24,10 +24,10 @@ Events are sent from Dashboard, and App can listen to them.
 
 With events, the App can react, e.g., to theme changes or receive a token with a handshake event.
 
-You can find all events [here](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md#events).
+You can find all events [here](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.mdx#events).
```

Comment:
Same as in current docs - the link no longer exists

Reply:
@krzysztofwolski done